nifi-nar-bundles/nifi-prometheus-bundle/nifi-prometheus-nar/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,46 @@
+nifi-prometheus-nar
```

Comment:
At a glance it looks like this NOTICE file will need to be updated.

Reply:
Hi @kevdoran, working on SSL support for the endpoint.